app/controllers/dashboard_controller.rb

```diff
@@ -17,6 +21,24 @@ def index
     end
   end
 
+  def autoyast
```

Comment:
https://trello.com/c/4pDXbtUV/81-create-and-maintain-api-documentation-for-velum

so maybe we could do that for all endpoints at a later time